odra-casper/test-env/src/env.rs

```diff
@@ -341,6 +346,9 @@ fn parse_error(err: engine_state::Error) -> OdraError {
             CasperExecutionError::NoSuchMethod(name) => {
                 OdraError::VmError(VmError::NoSuchMethod(name))
             }
+            CasperExecutionError::Revert(ApiError::Mint(0)) => {
+                OdraError::VmError(VmError::BalanceExceeded)
+            }
```

Comment:
What is it?

Reply:
It's an error thrown by the `getter-proxy` in:
```rust
transfer_from_purse_to_purse(main_purse, cargo_purse, amount, None).unwrap_or_revert();
```